python/eos/analysis.py

```diff
@@ -155,10 +155,22 @@ def __init__(self, priors, likelihood, global_options={}, manual_constraints={},
         for n in varied_parameter_names - used_parameter_names:
             eos.warn('likelihood does not depend on parameter \'{}\'; remove from prior or check options!'.format(n))
 
+
+    def __x_to_par(self, x):
```

Comment:
Following @pluegh's comment in the other PR, let's use a single leading underscore? Sorry for the flip-flopping...